### ISSUE TITLE
Performance issues with ngbRunTransition within ngbCollapse

### ISSUE THREAD

**Issue by Disane87** [ng-bootstrap/ng-bootstrap#3954]
### Bug description:
Actually we are experiencing significant performance impacts by your collapse module.
We have on site with many cards in it which are collapseable by your directive:

```html
<div class="card {{bgClass}} {{borderClass}} {{ accentClass }} {{cardClass}} mb-0 overflow-hidden w-100"
  *ngIf="showBody || showList" [style.height]="height">
  <!-- Header -->
  <div
    class="card-header validation d-flex flex-row justify-content-between {{bgClass}} {{borderClass}} {{headerBorderClass}} {{ headerClickToCollapse ? 'cursor-pointer' : '' }}"
    *ngIf="showHeader" (click)="headerClickToCollapse && toggleCollapseState($event)"
    [class.border-bottom-0]="isCollapsed">
    {{ headerText }}
    <div #contentWrapperHeader class="w-100">
      <ng-content select="[card-header]"></ng-content>
    </div>
    <ul class="nav nav-{{navType}} card-header-{{navType}}" *ngIf="navigationItems">
      <li class="nav-item" *ngFor="let item of listGroupItems">
        <a class="nav-link {{item.state}}" href="{{item.link}}">{{item.text}}</a>
      </li>
    </ul>
    <div class="card-header-actions">
      <ng-content select="[card-header-actions]"></ng-content>
      <a class="card-header-action" *ngIf="collapsable" (click)="toggleCollapseState($event)" aria-expanded="true">
        <vng-controls-icon *ngIf="isCollapsed" icon="chevron-down"></vng-controls-icon>
        <vng-controls-icon *ngIf="!isCollapsed" icon="chevron-up"></vng-controls-icon>
      </a>

    </div>
  </div>

  <!-- Body -->
  <div class="card-body animated {{customClassBody}} p-3" id="collapseBasic" [class.border-top-0]="isCollapsed"
    [(ngbCollapse)]="isCollapsed && collapsable" *ngIf="showBody" [animation]="false">
    <span #contentWrapperBody>
      <ng-content select="[card-body]"></ng-content>
    </span>
  </div>
</div>
```

Because we've noticed some serious issues on performance we've investigated the overall performance via chrome performance monitor.

We can clearly see the transitions are significantly slower than other things.
Since version 8.0.0 there is an animation for the components. These animations are already turned off but the problem persists:

![image](https://user-images.githubusercontent.com/956158/104886127-87388b00-5969-11eb-923b-f18d7989cfa0.png)

Why are these transitions executed when we've deactivated them? How can we tunr them completely off?

### Link to minimally-working StackBlitz that reproduces the issue:
I've already tried to reproduce that issue within stackblitz but I didn't managed to reprocude this issue.

You can fork a StackBlitz from one of our [demos](https://ng-bootstrap.github.io/#/components) and use it as a starting point.

Please note that we can _not_ act on bug reports without a _minimal_ reproduction scenario in a StackBlitz. Here is why:
https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 11.0.5

ng-bootstrap: 8.0.1

Bootstrap: 4.5.3

We have attached the per profile

[ngb-collapse-perf-issues.zip](https://github.com/ng-bootstrap/ng-bootstrap/files/5828774/ngb-collapse-perf-issues.zip)

**Comment by maxokorokov** [comment]
This will be fixed in `9.0.1` and `8.0.3` coming in the next couple of days.

**Comment by maxokorokov** [comment]
Fixed in `8.0.3` and `9.0.1`
